.github/workflows/daily-e2e-tests.yml

```diff
@@ -16,11 +16,11 @@ jobs:
           # - 9.0.0
           # - 9.1.0
 
-          # 9.2.0 is the earliest version where things work
-          - 9.2.20
-          - 9.3.16
-          - 9.4.13
-          - 9.5.7
+          # 10.0.11 is the earliest version where things work
+          # - 9.2.20
+          # - 9.3.16
+          # - 9.4.13
+          # - 9.5.7
```

Comment:
do we have a GH issue to fix this?

Reply:
I edited [this one](https://github.com/grafana/oncall/issues/3962) to include fixing e2e tests against Grafana 9.x